proposals/1219-storing-megolm-keys-serverside.md

```diff
@@ -177,6 +177,42 @@ Error codes:
 
 - `M_NOT_FOUND`: No backup version has been created.
 
+##### `PUT /room_keys/version/{version}`
+
+Update information about the given version, or the current version if `{version}`
+is omitted. Only `auth_data` can be updated.
+
+Body parameters:
+
+- `algorithm` (string): Optional. Must be the same as in the body parameters for `GET
```

Comment:
AIUI, in REST, the contents of PUT are supposed to be the full object that is to be stored, so I think `algorithm` should be required.